resources/assets/sass/app.scss

```diff
@@ -1,4 +1,5 @@
 @import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap";
+@import "node_modules/highlight.js/styles/github";
```

Comment:
@fredsted Are you happy with this as default style? Not sure what you wanted to use here; being able to choose would be great but added a level of complexity (plus additional preference to have somewhere)

I personally prefer a dark style